atlas/pool.go

```
@@ -15,29 +16,54 @@
 				return
 			}
-			_, err = conn.Prep("attach database '" + options.MetaFilename + "' as atlas").Step()
-			if err != nil {
-				return
+
+			se := func(query string) {
+				st, _, err := conn.PrepareTransient(query)
+				if err != nil {
+					panic("Error preparing transient statement: " + err.Error())
+				}
+				defer st.Finalize()
```

golangci-lint (1.62.2)

Error return value of `st.Finalize` is not checked

(errcheck)